Usage with Angular 1.5

ma-zal opened the pull request:
Updated dependencies to allow usage with Angular 1.5.
Note: Tested with Angular 1.5.0.

ma-zal committed:
Update README.md

Add Installation section to README

ma-zal committed:
Update README.md

ma-zal committed:
Update bower dependencies

e0ipso commented:
Tests are failing. Do we know the reason?

e0ipso commented on bower.json:
```diff
@@ -6,11 +6,11 @@
   "dependencies"    : {
-    "angular"       : ">=1.2.0 <1.5.0",
-    "angular-route" : ">=1.2.0 <1.5.0"
+    "angular"       : ">=1.2.0",
```
Wouldn't this allow untested larger major versions to be installed?

simison commented:
I would love to get this tested up to 1.5.x tho, it seems like one of the last major versions before 2.x anyway.

ma-zal committed:
Merge pull request #1 from e0ipso/master

Sync with original repository.

ma-zal committed:
Update NodeJS to 6.8 in .travis.yml

ma-zal committed:
Add Mocha to karma.json

ma-zal committed:
Update package.json

ma-zal committed:
Add serve-static to package.json

ma-zal committed:
Update test.server.js

ma-zal commented:
I fixed the tests. Problem was in too old testing packages. After update, all looks perfect.

ma-zal committed:
Limit dependencies to Angular 1.5.x

Do not expect automatically, that the future Angular 1.6.x will be fully compatible. So limit it to Angular 1.5.x for now.

ma-zal commented:
It would be nice, if new release of message-center (increase of version number) will occures in Bower after close of this pull request. :)

e0ipso commented on bower.json:
```diff
@@ -6,11 +6,11 @@
   "dependencies"    : {
-    "angular"       : ">=1.2.0 <1.5.0",
-    "angular-route" : ">=1.2.0 <1.5.0"
+    "angular"       : ">=1.2.0 <1.6.0",
```
@ma-zal can we drop the `<1.6.0` safely?

simison replied:
Only if you actually run tests with this: https://github.com/angular/angular.js/releases/tag/v1.6.0-rc.0

Otherwise it's unsafe.

simison commented on test/test.server.js:
```diff
@@ -1,14 +1,15 @@
 var minifyRequired = !!process.env.MINIFY;
 
 var connect = require('connect');
+var serveStatic = require('serve-static')
```
Missing `;`

e0ipso commented:
Fixed. Thanks all!

simison commented:
:tada: thanks @ma-zal & @e0ipso !
